scripts/build.sh

```diff
@@ -3,110 +3,178 @@
 ################################################
 # Build script for sparkswapd
 #
-# Params:
-# - EXTERNAL_ADDRESS (optional, for hosted daemons)
+# Options:
```

Comment:
I think you need to update the `env-setup` and `install.sh` scripts with these new options and make sure everything still works together